### DISCUSSION TITLE
iOS:excluded architecture是否有移除arm64选项的计划

### DISCUSSION THREAD

**Discussion by MrYu4** [Tencent/libpag#2177] [Q&A] [answered]
目前库编译依然针对模拟器存在限制arm64的情况
<img width="478" alt="Snipaste_2024-03-06_14-09-44" src="https://github.com/Tencent/libpag/assets/22674880/0da9d230-3627-467f-b25e-adc24b1164fe">
但是根据苹果的[相关文档](https://developer.apple.com/documentation/technotes/tn3117-resolving-build-errors-for-apple-silicon/#Update-pre-compiled-libraries-from-vendors)，这并不是一个长久的处理办法，能否根据文章中的建议将库重建为 XCFramework，并支持 Apple 芯片上的模拟器呢？

**Comment by domchen** [comment]
你看下release页面发布的编译好的库，一直都是xcframework而且一直支持Apple芯片上的模拟器：https://github.com/Tencent/libpag/releases/tag/v4.3.51 ，因为我们自己开发机都是Apple芯片，你截图的限制是哪里来的？

**Reply by shixiaoda** [reply]
podspec 里面有这一行
s.user_target_xcconfig = {
'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64'
}

**Reply by kevingpqi123** [reply]
感谢反馈，我们近几天处理下

**Reply by kevingpqi123** [reply, marked as answer]
podspec 中已移除，可以升级至 4.3.56 看下

**Comment by MrYu4** [comment]
> podspec 中已移除，可以升级至 4.3.56 看下

this work for me
